make/common.mk

```diff
@@ -50,7 +55,7 @@ GEN_JAVA_DIR = $(BUILD_DIR)/java
 endif
 
 ifndef J2OBJC_ARCHS
-J2OBJC_ARCHS = macosx iphone iphone64 watchv7k simulator simulator64
+J2OBJC_ARCHS = macosx iphone iphone64 watchv7k simulator simulator64 appletvos appletvsimulator
```

Comment:
Normally our focus has been on iOS builds with the macosx included mainly for testing purposes.
@tomball What are your thoughts on supporting Apple TV apps?